notebooks/time-to-merge-prediction/workshop/00_read_env.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>can we do this programatically? Because then we can prevent situations like people forgeting to make it a string</p><p>Can we have a if statement something like </p><p>if CHOSEN_MODEL not in ['rf', 'xgbc', 'svc', 'gnb']:</p><p>			print("Make sure your chosen model falls into the list of models, ('rf', 'xgbc', 'svc', 'gnb')")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>